packages/fx-core/src/plugins/solution/fx-solution/v2/provision.ts

```diff
@@ -187,7 +187,7 @@ export async function provisionResource(
     }
   }
 
-  if (isV2() && !isAzureProject(azureSolutionSettings)) {
+  if (isV2() && isAzureProject(azureSolutionSettings)) {
     solutionInputs.remoteTeamsAppId =
```

Comment:
how about spfx? no need for remote teams app id?

Reply:
Yeah. When multiple env is enabled, solution doesn't need to remember remote teams app id any more.